packages/notifier/README.md

```diff
@@ -1,48 +1,59 @@
-     failure. The reason is typically an error object, and should be, but may
-     in fact be any JavaScript value.
-
-The call `makeNotifierKit()` makes an `{updater, notifier}` pair.
-The call `makeSubscriptionKit()` for makes a similar
-`{publication, subscription}` pair. The first element of each pair (`updater`,
-`publication`) is used to produce the async iteration which can then be
-consumed using the second element of each pair (`notifier`,`subscription`).
-These have much API in common. The `notifier` and `subscription` both implement
-the JavaScript `AsyncIterable` API for consuming the iteration.
-
-There is no JavaScript standard for only producing an iteration, so this
-package defines the `IterationObserver` API. Both `updater` and `publication`
-implement the `IterationObserver` API. For both pairs, the `IterationObserver`
-provides *only* the authority to produce the iteration. The `AsyncIterable`
-provides *only* the authority to consume the iteration.
+An *async iteration* is an abstract sequence of values. It consists of any number of *non-final values* 
+in a fully ordered sequence revealed asynchronously over time. In other words, the values are fully 
+ordered compared to the time when they were provided to the publishing mechanism. In the case of 
+more than one publisher supplying values to the same sequence, the order is determined by when the 
+sequence receives the values. The point is that all consumers see the whole sequence, or a subset of 
+it, in the same order. 
+
+The sequence may continue indefinitely or may terminate in one of two ways:
+
+  * *Finish*: The async iteration successfully completes and reports a final *completion value*, 
+    Which can be any JavaScript value.
+  * *Fail*: The async iteration fails and reports an alleged final *reason*. This should be an error  
+     object, but can be any JavaScript value.
+   
+`makeNotifierKit()` makes an `{updater, notifier}` pair, while `makeSubscriptionKit()` makes a 
+similar `{publication, subscription}` pair. Each pair’s first element (`updater` or `publication`) 
+produces the async iteration which is then consumed using each pair’s second element (`notifier` 
+or `subscription`).
+
+`notifier` and `subscription` both implement the JavaScript `AsyncIterable` API to consume the 
+iteration. Both `updater` and `publication` implement the `IterationObserver` API, as defined in 
+this package (JavaScript has no standard for producing iterations). For both pairs:
+`IterationObserver` provides *only* the ability to produce the iteration. 
+`AsyncIterable` provides *only* the ability to consume the iteration.
 
 ## Example
 
-Although the JavaScript `AsyncIterable` API can be used directly, there are two
-more convenient ways to use it: Using the JavaScript `for-await-of` syntax and
-using the `observeIteration` adaptor.
+Let’s look at an example using `makeSubscriptionKit()` There are three “characters”; Paula the 
```

Comment:
Yeah, it was confusing here because the original had Alice as a subscriber and Bob as a consumer (repeated later on as well) despite both A&B using makeSubscriptionKit.